src/util.ts

```diff
@@ -280,3 +297,36 @@ export const addHashToBaseline = (hash: string, filepath: string): void => {
     ignoreMessages: baselineErrorsFile.meta.ignoreMessages
   })
 }
+
+export const toGitLabOutputFormat = (
+  errorSummaryMap: ErrorSummaryMap,
+  specificErrorMap: SpecificErrorsMap,
+  errorOptions: ErrorOptions
+): string => {
+  const result: GitLabErrorFormat[] = []
+
+  for (const [key, error] of errorSummaryMap.entries()) {
+    const specificErrors: SpecificError[] = getSpecificErrorsMatchingSummary(
+      error,
+      specificErrorMap,
+      errorOptions
+    )
+
+    specificErrors.forEach((specificError: SpecificError, index: number) => {
+      result.push(<GitLabErrorFormat>{
+        description: specificError.message,
+        check_name: 'typescript-errors',
+        fingerprint: `${key}-${index}`,
```

Comment:
Nice. I thought about adding the `line` here, but that would not work, since there can be multiple errors per line.
This works fine :+1: